src/js/wizards/Buy/1_Trade/index.jsx

```diff
@@ -56,11 +56,22 @@ class Trade extends Component {
   }
 
   validate(currencyQuantity, assetQuantity) {
-    if (currencyQuantity < 0 || assetQuantity < 0 || assetQuantity > (this.props.sellerBalance || MAX)) {
-      this.props.footer.disableNext();
-      this.setState({disabled: true});
-      return;
+    const limitless = this.props.offer.limitL === '0' && this.props.offer.limitH === '0';
+    if(limitless){
+      if (currencyQuantity < 0 || assetQuantity < 0 || assetQuantity > (this.props.sellerBalance || MAX)) {
```

Comment:
Don't know if we should disable on quantity higher than seller balance? Could still open and then let seller restock. What do you think?